source/fundamentals/transactions.txt

```diff
@@ -96,16 +96,13 @@ these methods:
          this method, see the :manual:`commitTransaction() page
          </reference/method/Session.commitTransaction/>`  in the Server manual.
          
-       .. note:: Retrying a Transaction
-          
-          The ``CommitTransaction()`` method is an idempotent function, which
-          means that you can attempt to commit a transaction multiple times
-          without changing data after the first successful commit.
-          
-          A transaction can succeed but return an error with the
-          ``UnknownTransactionCommitResult`` label. If you rerun the
-          ``CommitTransaction()`` method after receiving this error,
-          your data is not changed by the repeat attempts.
+      |  The ``CommitTransaction()`` method is an idempotent function, which
+         means that you can attempt to commit a transaction multiple times
+         without changing data after the first successful commit.
+         A transaction can succeed but return an error with the
+         ``UnknownTransactionCommitResult`` label. If you rerun the
+         ``CommitTransaction()`` method after receiving this error,
+         your data is not changed by the repeat attempts.
```

Comment:
I: I think there is an error with the indentation or spacing here as it's not appearing in the staging